packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -56,6 +89,16 @@ export default class SquirrelWindowsTarget extends Target {
       packager: this.packager,
     })
 
+    if (this.options.msi) {
+      await packager.info.callArtifactBuildCompleted({
+        file: artifactPath.replace(".exe", ".msi"),
+        target: this,
+        arch,
+        safeArtifactName: `${sanitizedName}-Setup-${version}${getArchSuffix(arch)}.msi`,
+        packager: this.packager,
+      })
+    }
```

Comment:
Just curious, what's the new functionality with the `msi` signing? I see that in a few places, like an additional signing step `L80-L82` and this `callArtifactBuildCompleted`

Reply:
An MSI is an executable file that needs to be signed; otherwise, running the installation will trigger security warnings. It only needs to be signed once. Even if the file name is changed later, there will be no issues, and re-signing is not necessary.